tools/proto-js-plugin/src/main/java/io/spine/js/generate/CodeLines.java

```diff
@@ -42,7 +43,7 @@
 @SuppressWarnings({"DuplicateStringLiteralInspection"
         /* The generated code duplicates the code used in test that checks it. */,
         "ClassWithTooManyMethods" /* A lot of simple utility methods for code generation. */})
-public final class JsOutput {
+public final class CodeLines {
```

Comment:
I think, what we have to do here now is to separate code construction from printing. I'd suggest that the `CodeLines` can produce an immutable object which is able to print the code into a string, file, stream, etc., depending on the plugin needs.